recipe/meta.yaml

```diff
@@ -11,15 +12,16 @@ source:
   patches:
     # See spyder-ide/spyder#8316
     - osx-zmq.patch
+    - spyder-base.patch
 
 build:
-  number: 3
+  number: {{ build }}
   entry_points:
     - spyder = spyder.app.start:main
   osx_is_app: true
```

Comment:
Perhaps this should be moved to the spyder subpackage?

Reply:
This tells Anaconda to make the package an application in Anaconda Navigator.

Reply:
I think the sounder-base sub package is the correct location but @ccordoba12 should confirm